platform-test.mak

```diff
@@ -16,9 +16,13 @@ UPGRADE_REF ?= $(shell git rev-parse --abbrev-ref HEAD)
 
 clean: clean-app clean-infra
 
+# The git branch to checkout
+# Set to `main` by default
+BRANCH ?= main
```

Comment:
optional nit: maybe name this something like `TEMPLATE_BRANCH`, since it could potentially be confused for the platform-test branch without looking at the code below. similar optional nit on the comment e.g.

```suggestion
# The template git branch to install
# Set to `main` by default
TEMPLATE_BRANCH ?= main
```